File: courses/ADO.NET.DAL/models/Course.cs
namespace ADO.NET.DAL.models;

public class Course
{
    public int Id { get; set; }
    public string? Name { get; set; }
    public string? Description { get; set; }
    public ICollection<CourseUser> CourseUsers { get; set; } = new List<CourseUser>();
}

File: Java/courses/src/main/java/ua/edu/chnu/courses/models/Course.java
package ua.edu.chnu.courses.models;

import jakarta.persistence.*;
import lombok.*;

@NoArgsConstructor
@EqualsAndHashCode
@Entity
@Table(name = "courses")
public class Course {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Getter
    @Setter
    @Column(nullable = false)
    private String name;

    @Getter
    @Setter
    private String description;

    public Course(String name, String description) {
        this.name = name;
        this.description = description;
    }
}